src/accounts/accounts.controller.ts

```diff
@@ -240,4 +245,19 @@ export class AccountsController {
     const result = await this.accountsService.modifyPermissions(params);
     return handleServiceResult(result);
   }
+
+  @ApiOperation({
+    summary: "Get chain's treasury Account",
+    description: 'This endpoint retrieves treasury Account details',
```

Comment:
```suggestion
    description: 'This endpoint retrieves the treasury Account details',
```

I think its singular, otherwise use the plural form